apps/api/src/domain/games/memory-match/queries/memory-match-queries.fetch-many.e2e.spec.ts

```diff
@@ -64,37 +127,9 @@ describe(`when querying for a memory match round: fetch many`, () => {
      * TODO[https://coscrad.atlassian.net/browse/CWEBJIRA-305]
      * Return unpublished rounds to admin users.
      */
-    describe(`when the user is unauthenticated`, () => {
+    describe(`when the user is not-authenticated (public queries)`, () => {
```

Comment:
`unauthenticated` is a technical term. Is there a reason for using `not-authenticated`? It's no big deal, just wondering. I like the (public queries) part.

Reply:
This was in use in one of the files I used as a model, will change to unauthenticated.